command/ca/provisioner/remove.go

```diff
@@ -21,7 +22,11 @@ func removeCommand() cli.Command {
 			},
 			cli.StringFlag{
 				Name:  "kid",
-				Usage: "The <kid> (Key ID) of for the provisioner key to be removed.",
+				Usage: "The <kid> (Key ID) of for the JWK provisioner key to be removed.",
+			},
+			cli.StringFlag{
+				Name:  "client-id",
+				Usage: "The <id> (Client ID) of for the OIDC provisioner to be removed.",
```

Comment:
--all flag also cannot be used in conjunction with the --client-id flag. (update docs for --all)